//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// acdef_T_P.h
//
// Code generation for function 'acdef_T_P'
//

#ifndef ACDEF_T_P_H
#define ACDEF_T_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdef_T_P(double P_init, double V_init, double A_init, double P_wayp,
               double V_max, double A_min, double J_max, double J_min, double T,
               creal_T t[14]);

#endif
// End of code generation (acdef_T_P.h)
